Packages/ClientRuntime/Sources/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -11,9 +11,67 @@ import Darwin
 #endif
 
 public class CRTClientEngine: HttpClientEngine {
+    actor ConnectionPoolContainer {
```

Comment:
```suggestion
    actor ConnectionPoolManager {
```

It should be `ConnectonPoolManager` and there is no reason it to be nested class. Although, keep it internal.

Reply:
It is internal since it should only be used with the paren type. I also considered calling it `SerialExecutor` as the nested type because that is what it is.